de.metas.async/src/main/sql/postgresql/system/42-de.metas.async/5449521_sys_703_AddWorkpackageNotified.sql

```diff
@@ -0,0 +1,452 @@
+;
+
+-- 03.03.2016 17:17:15 OEZ
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Element (AD_Client_ID,AD_Element_ID,AD_Org_ID,ColumnName,Created,CreatedBy,EntityType,IsActive,Name,PrintName,Updated,UpdatedBy) VALUES (0,543030,0,'IsNotified',TO_TIMESTAMP('2016-03-03 17:17:15','YYYY-MM-DD HH24:MI:SS'),100,'de.metas.async','Y','Notified','Notified',TO_TIMESTAMP('2016-03-03 17:17:15','YYYY-MM-DD HH24:MI:SS'),100)
+;
+
+-- 03.03.2016 17:17:15 OEZ
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Element_Trl (AD_Language,AD_Element_ID, Description,Help,Name,PO_Description,PO_Help,PO_Name,PO_PrintName,PrintName, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy) SELECT l.AD_Language,t.AD_Element_ID, t.Description,t.Help,t.Name,t.PO_Description,t.PO_Help,t.PO_Name,t.PO_PrintName,t.PrintName, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy FROM AD_Language l, AD_Element t WHERE l.IsActive='Y' AND l.IsSystemLanguage='Y' AND l.IsBaseLanguage='N' AND t.AD_Element_ID=543030 AND NOT EXISTS (SELECT * FROM AD_Element_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Element_ID=t.AD_Element_ID)
+;
+
+-- 03.03.2016 17:17:25 OEZ
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Column (AD_Client_ID,AD_Column_ID,AD_Element_ID,AD_Org_ID,AD_Reference_ID,AD_Table_ID,AllowZoomTo,ColumnName,Created,CreatedBy,DefaultValue,EntityType,FieldLength,IsActive,IsAdvancedText,IsAllowLogging,IsAlwaysUpdateable,IsAutocomplete,IsCalculated,IsEncrypted,IsIdentifier,IsKey,IsLazyLoading,IsMandatory,IsParent,IsSelectionColumn,IsSyncDatabase,IsTranslated,IsUpdateable,Name,SeqNo,Updated,UpdatedBy,Version) VALUES (0,554191,543030,0,20,540749,'N','IsNotified',TO_TIMESTAMP('2016-03-03 17:17:25','YYYY-MM-DD HH24:MI:SS'),100,'N','de.metas.async',1,'Y','N','Y','N','N','N','N','N','N','N','N','N','N','N','N','Y','Notified',0,TO_TIMESTAMP('2016-03-03 17:17:25','YYYY-MM-DD HH24:MI:SS'),100,0)
+;
+
+-- 03.03.2016 17:17:26 OEZ
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Column_Trl (AD_Language,AD_Column_ID, Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy) SELECT l.AD_Language,t.AD_Column_ID, t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy FROM AD_Language l, AD_Column t WHERE l.IsActive='Y' AND l.IsSystemLanguage='Y' AND l.IsBaseLanguage='N' AND t.AD_Column_ID=554191 AND NOT EXISTS (SELECT * FROM AD_Column_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Column_ID=t.AD_Column_ID)
+;
+
+-- 03.03.2016 17:17:45 OEZ
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
```

Comment:
generally, please separate DDL from DML and add a `COMMIT;` between them, to avoid a possible error about "pending" trigger events when we apply migration scripts.